debian/control

```diff
@@ -34,6 +34,7 @@ Build-Depends: cmake (>= 3.10.2),
                libcunit1-dev,
                libcurl4-openssl-dev,
                libexpat1-dev,
+# Make-Check   libffi-dev [!amd64],
```

Comment:
i'd suggest amend the commit message to explain why we add `!amd64` here.

Reply:
Ok, the commit message has been modified.